packages/legend-query-builder/style/_query-builder.scss

```diff
@@ -833,17 +833,292 @@
     color: var(--color-white);
   }
 
+  &__table {
+    min-width: 50%;
+  }
+
+  &__item {
+    width: 100%;
+    border: 0.1rem solid var(--color-dark-grey-300);
+    border-radius: 0.3rem;
+    font-family: monospace;
+    font-size: 1.3rem;
+    padding: 1rem;
+    margin: 5px 0;
+    line-height: 1.8rem;
+
+    .decorationsOverviewRuler {
+      display: none !important;
+    }
+
+    &__label {
+      color: var(--color-white);
+    }
+  }
+}
+
+.query-builder__function-parameters-validation__container {
+  width: 95%;
+  margin: 1% auto 0;
+
+  &__icon svg {
+    margin-left: 0.5rem;
+    color: var(--color-light-grey-400) !important;
+  }
+
+  &__icon svg:hover {
+    color: var(--color-white);
+  }
+
+  &__table {
+    min-width: 50%;
+  }
+
+  &__item {
+    &__label {
+      color: var(--color-white);
+    }
+  }
+}
+
+.query-builder__allocation__container {
+  width: 95%;
+  margin: 1% auto 0;
+
+  &__icon svg {
+    margin-left: 0.5rem;
+    color: var(--color-light-grey-400) !important;
+  }
+
+  &__icon svg:hover {
+    color: var(--color-white);
+  }
+
+  &__table {
+    width: 100%;
+  }
+
+  &__info {
+    margin-bottom: 0.5em;
+    display: inline-block;
+
+    &__var-name {
+      display: inline-block;
+      text-indent: 5rem;
```

Comment:
Are the text-indents for aligning the detail values to the same line? 
<img width="176" alt="image" src="https://github.com/finos/legend-studio/assets/41248956/4acd2b1f-69aa-4782-99b4-7593fbfcbbfe">
Is this preferable @akphi, I'm not sure if its necessary to add the text indent like this, if you wanted it structured in same alignment I think you could also use a table so its column widths may be responsive?